### ISSUE TITLE
Does not behave well with Windows split panes

### ISSUE THREAD

**Issue by michael-imbeault** [Razumihin/obsidian-fullscreen-plugin#3]
Love the plugin - however sometimes I would want to focus on a pane I have open, but split it on my screen (put it on the half left side) while I have some other application split on the right side. With the plugin on, the Obsidian window doesn't want to leave fullscreen mode.

I would have expected the plugin to focus one particular note I have open, but to allow window size manipulation / location.

**Comment by Razumihin** [comment]
Yeah, there's quite a lot of limitations how the plugin manages the windows as it just requests the fullscreen mode from the layout management system. I hope I'll eventually have time to look into making the plugin handle different cases better, but i've lately been too busy with writing code that I get paid for 😉
